src/app/index.test.js

```diff
@@ -40,6 +40,179 @@ describe("Templates as a service... again!", () => {
         })
     })
 
+      it("should return a govukTemplate", () => {
+          const expected = `<!DOCTYPE html>
```

Comment:
Can we move this into a fixture file please?  Or use jest's snapshots if you're just wanting to know this hasn't changed.

Reply:
That's not the pattern followed by the other tests in this suite. Should we change them too?